### ISSUE TITLE
qtcreator projects fail to link

### ISSUE THREAD

**Issue by arturoc** [openframeworks/openFrameworks#6420]

**Comment by ofTheo** [comment]
@thomasgeissl is the linking issue with qtcreator just that dylib one or something else?

**Comment by thomasgeissl** [comment]
it is just libfmodex.dylib for me.

**Comment by ofTheo** [comment]
@thomasgeissl any chance you could do a PR with a fix ( if you figured it out )? 
I think this is one of the last remaining issues before we are good to publish the next release.

**Comment by ofTheo** [comment]
Okay after a couple of string errors with the wizard I can get qtCreator to build. 
I see a error with libfmodex.dylib not being found @execuable_path but I think that is because the dylib is being copied correctly to the app bundle while the executable is being left in the bin/ folder

See:
<img width="826" alt="image" src="https://user-images.githubusercontent.com/144000/68997347-3eafaa00-085a-11ea-8551-f8151dc1e208.png">

If I manually copy over the executable into AppName.app/Content/MacOS/ then I can launch the app fine from the Finder. 

So not sure if it just needs to be set to be a non-console app or if there is something screwy going on with the output location?

**Comment by ofTheo** [comment]
Yup - that is it I think. 

If you do:
`
    // create a console window when the application start
    consoleApplication: false
`

Apps build and run fine.

**Comment by ofTheo** [comment]
Hmm - I don't see this issue on Ubuntu with the nightly. 
Just tried and could run a couple of different examples created by the qtcreator wizard. 
No linker errors for me. This is running the noWindowExample. 

<img width="1378" alt="image" src="https://user-images.githubusercontent.com/144000/69682560-9de59980-1066-11ea-9e07-c8ee2b9855fd.png">

**Comment by ofTheo** [comment]
@arturoc I think this isn't an issue anymore with the nightly builds. I have been using qtcreator to do builds without issues. Maybe the other qtcreator fixes fixed it? Going to close it. Def re-open if it still happens for you.

**Comment by arturoc** [comment]
yes it's working for me fine in a new installation but in my laptop the same version of qtcreator doesn't work. sometimes the settings for the compiler break somethings but the same project is compiling fine with makefiles in that install so want to make sure it's just some error in this specific install and not something it's going happen to more people
